nolooking/umbrel-app.yml

```diff
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: nolooking
+category: Lightning Node Management 
+name: nolooking
+version: "0.1.0"
+tagline: Use PayJoin to open channels for your lightning node.
+description: >-
+  Nolooking leverages Pay-to-Endpoint (payjoin) to negotiate a channel open for your lightning node, initiated by any BIP78 supporting wallet.
+
+  You can open multiple lightning nodes in a single payment, improving privacy, saving on transaction fees and wait time.
+
+  Read the article about [Lightning Powered PayJoin](https://chaincase.app/words/lightning-payjoin?ref=github) to hear all the benefits!
+
+  Please note that nolooking is still in early development. By using nolooking, you hold all responsibility for any loss of funds or other grievances which may arise. We are rapidly iterating on this product, feedback and contributions are strongly appreciated.
+releaseNotes: >-
+  v0.1.0 -- Initial Halloween alpha release - many more features to come!
+developer: nolooking
+website: https://nolooking.chaincase.app
+dependencies:
+  - lightning
+repo: https://github.com/chaincase-app/nolooking
+support: https://github.com/chaincase-app/nolooking/issues
+port: 4444
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: "/pj"
+defaultUsername: ""
+defaultPassword: ""
+submitter: Nick Farrow
```

Comment:
@nickfarrow do you happen to be affiliated with the nolooking project? If so, shall we use `nolooking` as the submitter then? We recommend keeping the submitter name same as the developer name if the app is packaged by the app's organization or one of its developers. This ensures maximum confidence from users that the app is coming straight from its source.

Reply:
Sure am! (you can find my handle @utxoclub on [nolooking.chaincase.app.](https://nolooking.chaincase.app)). Agreed, that would be preferrable. Making that change. We also intend to get future images located on the chaincase dockerhub (instead of satsale)